site/docs/resources/faq.md

```diff
@@ -13,45 +13,43 @@ It reached the first stable version a few weeks later.
 
 ## How is grammY developed?
 
-grammY is completely free and open-source software, developed by a team of volunteers.
+grammY is a completely free and open-source software, developed by a team of volunteers.
 Its code is available on GitHub.
 
 You're welcome to [join us](https://t.me/grammyjs)!
 
 ## What programming language does grammY use?
 
-grammY is written from the ground up in TypeScript, a superset of JavaScript.
-It can therefore run on Node.js.
-However, grammY can also run on Deno, which positions itself as the successor of Node.js.
-(Technically, you can actually run grammY in modern browsers, even though this will rarely be useful.)
+grammY is written from the ground up in TypeScrip &mdash; a superset of JavaScript. Therefore, it runs on Node.js.
```

Comment:
This is incorrect typography in English. Either revert, or, if you need to see a dash here, use
```suggestion
grammY is written from the ground up in TypeScript—a superset of JavaScript. Therefore, it runs on Node.js.
```
We're also trying to keep HTML usage as low as possible.